modules/bar/media/index.ts

```diff
@@ -65,6 +65,7 @@ const Media = () => {
                         Widget.Label({
                             class_name: "bar-button-icon media",
                             label: songIcon.bind("value"),
+                            visible: show_label.bind("value"),
```

Comment:
```suggestion
                            label: songIcon.bind("value").as(v => v || "󰝚"),
```
We'll handle drawing the music note icon here if the player doesn't generate an icon itself (fallback icon). And since this Widget.Label handles the icon at all times, we don't need to toggle visibility.